jbpm-wb-case-mgmt/jbpm-wb-case-mgmt-client/src/main/java/org/jbpm/workbench/cm/client/roles/CaseRoleItemView.java

```diff
@@ -96,4 +220,59 @@ public void setLastElementStyle(boolean last){
         }
     }
 
+    public void showMoreItems(Div lineDiv,UnorderedList roleList,Div moreItemsLinkDiv,Div lessItemsLinkDiv) {
+        removeCSSClass(lineDiv,"kie-roles-item-line");
+        addCSSClass(lineDiv,"kie-roles-item-line-expanded");
+        addCSSClass(roleList,"popover-content");
+        addCSSClass(moreItemsLinkDiv,"hidden");
+        removeCSSClass(lessItemsLinkDiv,"hidden");
+    }
+
+    public void showLessItems(Div lineDiv,UnorderedList roleList,Div moreItemsLinkDiv,Div lessItemsLinkDiv) {
+        addCSSClass(lineDiv,"kie-roles-item-line");
+        removeCSSClass(lineDiv,"kie-roles-item-line-expanded");
+        removeCSSClass(roleList,"popover-content");
+        if(!isOwnerAssignment()){
+            removeCSSClass(moreItemsLinkDiv,"hidden");
+        }
+        addCSSClass(lessItemsLinkDiv,"hidden");
+    }
+
+    @EventHandler("more-users")
+    @SuppressWarnings("unsued")
+    public void onMoreUsers(@ForEvent("click") final org.jboss.errai.common.client.dom.Event event) {
+        roleAssignmentListOpenEventEvent.fire(new CaseRoleAssignmentListOpenEvent(roleName.getTextContent() + USERS_LINE));
+    }
+
+    @EventHandler("less-users")
+    @SuppressWarnings("unsued")
+    public void onLessUsers(@ForEvent("click") final org.jboss.errai.common.client.dom.Event event) {
+        showLessItems(usersLineDiv,roleUsers,moreUsersLinkDiv,lessUsersLinkDiv);
+    }
+
+    @EventHandler("more-groups")
+    @SuppressWarnings("unsued")
+    public void onMoreGroups(@ForEvent("click") final org.jboss.errai.common.client.dom.Event event) {
+        roleAssignmentListOpenEventEvent.fire(new CaseRoleAssignmentListOpenEvent(roleName.getTextContent() + GROUPS_LINE));
+    }
+
+    @EventHandler("less-groups")
+    @SuppressWarnings("unsued")
+    public void onLessGroups(@ForEvent("click") final org.jboss.errai.common.client.dom.Event event) {
+        showLessItems(groupsLineDiv,roleGroups,moreGroupsLinkDiv,lessGroupsLinkDiv);
+    }
+
+    public void onCaseRoleAssignmentLineOpenEvent(@Observes CaseRoleAssignmentListOpenEvent event) {
```

Comment:
@cristianonicolai  the idea was all the displayed CaseRoleItem consume the event and close themselves it they are not the one who launched the event. If I call a simple method only the one who is calling notice the change. Make it sense?